config/README.rst

```diff
@@ -94,8 +94,38 @@ After running the script you should manually do the following steps:
 4. Check in possible changes in the zopefoundation/meta repository.
 
 
+Options
++++++++
+
+It is possible to configure a deliberately small set of options a `.meta.cfg`
+inside the package repository. This file also stores the template name and
+commit id of the *meta* repository at the time of the run.
+
+  .. code-block:: ini
+
+    [meta]
```

Comment:
There should be a comment where this `.meta.cfg` file is generated from.

Reply:
I added code to include this hint. I would like to omit it here in the documentation.